docs/gathered-data.md

```diff
@@ -660,6 +660,15 @@ Docs for relevant types: https://pkg.go.dev/github.com/openshift/api/operatorcon
   * 4.8+
 
 
+## PodDefinition
+
+collects pod definition from pods that are firing one of the configured alerts.
```

Comment:
What are the 'configured alerts'? How do I look them up / where do I find the 'configuration'?

Reply:
The configuration alerts are currently hardcoded in the IO. We are working to get it from a service that we designed [here](https://github.com/RedHatInsights/insights-operator-gathering-conditions-service) and the conditions are [here](https://github.com/RedHatInsights/insights-operator-gathering-conditions).

Reply:
I think the point here is to extend/update the documentation. I think it would be good to address it in a new task (to document new config, conditions source etc. once the new service is in prod)